spec/factories/statements.rb

```diff
@@ -49,5 +49,11 @@
       year { 2025 }
       month { 9 }
     end
+
+    trait :with_milestones do
+      after :create do |statement|
+        create_list(:milestones_statement, 2, statement:)
```

Comment:
I think this will create 2 separate miletones of type `started` ? Might need to be 2 separate `create`s